src/Linux/Native/RunAsUser/readme.txt

```diff
@@ -0,0 +1,11 @@
+========================================================================
+    MAKEFILE PROJECT : Microsoft.R.Host.RunAsUser Project Overview
+========================================================================
+
+Build instructions:
+Compile: g++ -std=c++14 -fexceptions -fpermissive -O0 -ggdb -I../src -I../lib/picojson -c ../src/*.c*
+Link: g++ -g -o Microsoft.R.Host.RunAsUser.out ./*.o -lpthread -L/usr/lib/x86_64-linux-gnu -lpam
```

Comment:
The output binary shouldn't have an extension - `a.out` is a very old convention that's no longer in use. So this should just be `Microsoft.R.Host.RunAsUser`.